jenkins/databricks/test.sh

```diff
@@ -38,21 +39,32 @@ CUDF_UDF_TEST_ARGS="--conf spark.python.daemon.module=rapids.daemon_databricks \
     --conf spark.rapids.python.memory.gpu.allocFraction=0.1 \
     --conf spark.rapids.python.concurrentPythonWorkers=2"
 
+## '--conf spark.xxx.xxx=xxx' to 'export PYSP_TEST_spark_xxx_xxx=xxx'
+if [ -n "$SPARK_CONF" ]; then
```

Comment:
We need to translate `--conf spark.xxx=xxx` into `export PYSP_TEST_spark_xxx=xxx`, as we are calling the script `run_pyspark_from_build.sh` to parallelly run the integration tests. 

The scripts `run_pyspark_from_build.sh`  requires us to translate spark confs into `export PYSP_TEST_spark_xxx=xxx` to run IT with parallelism.